src/components/data-table-v2/_data-table-v2-expandable.scss

```diff
@@ -31,19 +40,34 @@
     }
 
     + tr[data-child-row] {
+      display: table-row;
       td {
         border-top: 0;
         padding-bottom: $spacing-xs;
+        max-height: 100%;
+        opacity: 1;
+        transition: all $transition--expansion $carbon--standard-easing;
+
+        &:before {
+          content: '';
+          position: absolute;
+          top: 0;
+          left: 0;
+          height: 100%;
+          width: 2px;
```

Comment:
Can we switch this to `width: rem(2px);`

Reply:
Good catch! thanks!